docs/style/scala.md

```diff
@@ -0,0 +1,106 @@
+Raster Foundry Style Guide
+============
+
+Comments
+--------
+
+Single line comments should be wrapped in `/**...*/`.
+
+```scala
+/** good */
+// bad
```

Comment:
This is only true for ScalaDoc comments. So comments above methods, classes, traits, objects, and packages that you would want a ScalaDoc process (e.g. https://geotrellis.github.io/scaladocs/latest/#geotrellis.raster.ArrayTile)

Otherwise, `//` is totally fine for inline comments that aren't ScalaDoc targeted